push_notifications/gcm.py

```diff
@@ -201,7 +245,38 @@ def gcm_send_bulk_message(registration_ids, data, **kwargs):
 		if len(registration_ids) > max_recipients:
 			ret = []
 			for chunk in _chunks(registration_ids, max_recipients):
-				ret.append(_gcm_send_json(chunk, data, **kwargs))
+				ret.append(_cm_send_json(chunk, data, cloud_type="GCM", **kwargs))
 			return ret
 
-	return _gcm_send_json(registration_ids, data, **kwargs)
+	return _cm_send_json(registration_ids, data, cloud_type="GCM", **kwargs)
+
+
+def fcm_send_bulk_message(registration_ids, data, **kwargs):
```

Comment:
There's no need, I separated to be explicit that configs e prepares for GCM and FCM are different.
If separated in different methods any custom properties or behaviors in FCM will be easier to do